8vim/src/main/res/layout/content_launcher.xml

```diff
@@ -20,13 +20,13 @@
                 android:id="@+id/constraintlayout1"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:padding="5dp">
+                android:padding="5sp">
```

Comment:
Every time you touch a file there are changes like these. This cannot be. If you want to refactor something or change some part of the code, make it a separate commit. Each commit should be purposeful and small. it should be easily explainable in a single sentence.
Take care of such things from next time.